koku/masu/database/ocp_report_db_accessor.py

```diff
@@ -596,6 +596,65 @@ def populate_monthly_tag_cost_sql(  # noqa: C901
         LOG.info(log_json(msg="populating monthly tag costs", context=ctx))
         self._prepare_and_execute_raw_sql_query(table_name, sql, sql_params, operation="INSERT")
 
+    def populate_node_hour_tag_cost_sql(
```

Comment:
Do we really need this function? It looks like a copy of `populate_monthly_tag_cost_sql` where the only difference is that we don't change the sql template based on the cost_type.

I would just drop `monthly` from `populate_monthly_tag_cost_sql` and just reuse this func when populating the hourly costs.